lib/LLVMIRCodeGen/CommandLine.cpp

```diff
@@ -108,3 +108,8 @@ llvm::cl::opt<bool> bundleAPIVerbose(
     "bundle-api-verbose",
     llvm::cl::desc("Print more details in the bundle API header file"),
     llvm::cl::init(false), llvm::cl::cat(bundleSaverCat));
+
+llvm::cl::list<std::string> bundleObjectsOpt(
+    "bundle-objects",
+    llvm::cl::desc("Option to add other external object files to the bundle"),
```

Comment:
Do you expect list of file names here? If so, maybe state it clearly.

Reply:
I changed the description to state that we expect a list of names.